docs/writing-tests/assumptions.md

```diff
@@ -4,7 +4,7 @@ The tests make a number of assumptions of the user agent, and new
 tests can freely rely on these assumptions being true:
 
  * The device is a full-color device.
- * The device has a viewport width of at least 800px.
+ * The device has viewport dimensions of at least 800px by 600px.
```

Comment:
Is this RFC material?

Reply:
I wouldn't know. I was hoping that given the discussion in #7642 this was just missing information. Not sure I want to make the time to write an RFC for this.

Reply:
I think it's probably OK? Although having said that I don't think we currently enforce a minimum window size in wptrunner, except for reftests, so maybe we should have a followup issue to actually implement this? In which case a small RFC wouldn't hurt I think.

Reply:
I filed https://github.com/web-platform-tests/rfcs/issues/188 as follow-up. Is that sufficient to land this change? I primarily care about getting this test fixed, but it seems good to update the documentation as well as per the discussion many years ago.